lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -131,14 +311,25 @@ def createTransformAttributesSection(self, ufeSceneItem, sectionName, attrsToAdd
 
             # Then add any reamining Xformable attributes
             self.addControls(attrsToAdd)
-            self.addUfeTransform3dObserver()
 
-    def buildUI(self, ufeSceneItem):
+            # Add a custom control for UFE attribute changed.
+            t3dObs = UfeAttributesObserver(self.item)
+            self.defineCustom(t3dObs)
```

Comment:
Use the defineCustom which gets the create/replace method and calls editorTemplate.